### PR TITLE
ploigos-tool-java-11 init add

### PR DESCRIPTION
**Pull request by itewk**
# purpose

tl;dr - make java 11 the default for anything using java.

# details

* remove ploigos-tool-java-8
* create ploigos-tool-java
* create java 8 and java 11 flavors of the following, with java 11 being the default flavor for edge/latest/nightly
   - ploigos-tool-java
   - ploigos-tool-maven
   - ploigos-tool-jkube
   - ploigos-ci-agent-jenkins

# is it breaking?

in theory no. none of the ref pipelines reference the ploigos-tool-java-8 image directly, and the names for the images that are referenced arn't chaning. the only 'risk' is that now the default java version for the ploigos-tool-maven will be java8 rather then java11. but the java nerds tell me, "that should only break really bad apps"...sooo....hold my beer.

# why java 8 and java 11?
cuz it doesn't really cost us any overhead to have it, we already had the automation in place in our build pipeline to be able to support this sort of matrix of things and I'd rather have the images and not need them then run into some app that needs them and not have them for such little cost over head. default will remain "whatever the latest GA/long term support UBI available java version is" which is currently 11, whenever 17 or whatever GAs, we will add that and make it the "default".

### CODE REVIEW

**Comment by itewk** [comment]
ping

File: ploigos-tool-java/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# ploigos-tool-java-8
```
Comment (dwinchell) [inline ploigos-tool-java/README.md:1]:
```suggestion
# ploigos-tool-java
```

File: ploigos-tool-java/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+This repository contains the container definition for creating the Ploigos workflow
+tool Java 8 image.
```
Comment (dwinchell) [inline ploigos-tool-java/README.md:4]:
```suggestion
tool Java image.
```

File: ploigos-tool-java/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+This container image is intended to be the base image for all other Ploigos workflow
+container images requiring Java 8.
```
Comment (dwinchell) [inline ploigos-tool-java/README.md:7]:
```suggestion
container images requiring Java.
```

**Review by dwinchell — CHANGES_REQUESTED** [review]

File: ploigos-tool-java/README.md (hunk 1 above)
Comment (itewk) [reply ploigos-tool-java/README.md]: done.

File: ploigos-tool-java/README.md (hunk 3 above)
Comment (itewk) [reply ploigos-tool-java/README.md]: done.

File: ploigos-tool-java/README.md (hunk 2 above)
Comment (itewk) [reply ploigos-tool-java/README.md]: done.

**Review by dwinchell — APPROVED** [review]
